### PR TITLE
feat(removeOrgMember):Added feature to delete user from organisation and teams

### PR DESCRIPTION
**Pull request by Aditya3034**
### Functionality:
Added feature to delete user from organisation and teams.

### Solution:
Changes in members-access component an implementation of method in tark of organization.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
1.Open emulator
2.Go to member access in my organization
3.you can see the delete button only when the the role is set as admin.
4.the delete button will delete the user from the organisation as well as from teams.

### CODE REVIEW

File: functions/model/organization/tark/removeOrgMember.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const p1 = updateMember(updateJson, orgDomain, remove);
+    if (!flag) {
+      getOrgRawData(orgDomain).then((rawData) => {
+        let totalNumberOfMembers = parseInt(rawData.TotalNumberOfMembers);
+        totalNumberOfMembers =totalNumberOfMembers- 1;
+        const updateRawDataInputJson = {
+          TotalNumberOfMembers: totalNumberOfMembers,
+        };
+        updateApplication(updateRawDataInputJson);
+        updateOrgRawData(updateRawDataInputJson, orgDomain);
```
Comment (sanjaykrishna1203) [inline functions/model/organization/tark/removeOrgMember.js:49]: You cannot update the application rawdata with organization raw data.

File: src/app/Interface/UserInterface.ts [hunk 2]
```diff
@@ -123,6 +123,7 @@ export interface MemberData{
     TeamManager: boolean;
     Teams: [];
     Email: string;
+    OrgMember: string[];
 }
```
Comment (sanjaykrishna1203) [inline src/app/Interface/UserInterface.ts:127]: what is the use of orgMember array in member data interface?

File: src/app/body/view-organization-details/members-access/members-access.component.ts [hunk 3]
```diff
@@ -25,10 +32,74 @@ import { UserServiceService } from 'src/app/services/user-service/user-service.s
...
+      this.organizationDomain = this.backendService.getOrganizationDomain();
+    }
+    callable({OrganizationDomain: this.organizationDomain, OrgMembers: this.membersEmail, Remove: remove}).subscribe({
+      next: (data) => {
+        this.getOrgMembers(this.organizationDomain);
+        this.enableLoader = false;
+        const index = this.membersEmail.indexOf(remove);
+        if (index != -1) {
+          this.membersEmail.splice(index, 1);
+          this.team.TeamMembers.splice(index, 1);
```
Comment (sanjaykrishna1203) [inline src/app/body/view-organization-details/members-access/members-access.component.ts:74]: this.team will be undefined and there is no need for this variable can be removed

File: src/app/body/view-organization-details/members-access/members-access.component.html [hunk 4]
```diff
@@ -36,7 +36,34 @@
...
+                        <div class="modal-content">
+                            <div class="modal-header">
+                                <h2 class="modal-title" id="DeleteModalLabel">Are you sure you want to delete this user?</h2>
+                                <span class="btn">
+                                    <mat-icon fontSet="material-icons-outlined" class="align-middle" data-dismiss="modal" aria-label="Close">close</mat-icon>
+                                </span>
+                            </div>
+                            <div class="row p-m-2 text-center">
+                                <div class="col" style="font-size: medium;flex-wrap: wrap;"><h4> The user will be deleted from everywhere including Teams and Organisation if you delete.</h4></div>
+                            </div>
```
Comment (sanjaykrishna1203) [inline src/app/body/view-organization-details/members-access/members-access.component.html:56]: Content can be improved
_Are you sure to delete the user? This Action will delete the user from all the teams the user is part of and cannot be undone._

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]
Please do the required changes

File: functions/model/organization/tark/removeOrgMember.js (hunk 1 above)
Comment (Aditya3034) [reply functions/model/organization/tark/removeOrgMember.js:49]: Shall I use another variable as inputJson to update the application rawdata

File: src/app/Interface/UserInterface.ts (hunk 2 above)
Comment (Aditya3034) [reply src/app/Interface/UserInterface.ts:127]: no there is no use as such now bhaiya I removed this property.

File: src/app/body/view-organization-details/members-access/members-access.component.html (hunk 4 above)
Comment (Aditya3034) [reply src/app/body/view-organization-details/members-access/members-access.component.html]: Did the required changes.

**Review by sanjaykrishna1203 — APPROVED** [review]

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**